### PR TITLE
Update prometheus rules

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
Signed-off-by: Piyush Nimbalkar <pnimbalkar@purestorage.com>

**What this PR does / why we need it**:
We usually update the prometheus rules regularly. The operator downloads the rules file from our docs website regularly. Updating the tests to match with the latest rules file.

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/storage/portworx/testspec/prometheusRule.yaml [hunk 1]
```diff
@@ -29,19 +29,6 @@ spec:
       labels:
         issue: Portworx volume {{$labels.volumeid}} usage on {{$labels.instance}} is critical.
         severity: warning
-    - alert: PortworxVolumeWillFill
```
Comment (haibinxie) [inline drivers/storage/portworx/testspec/prometheusRule.yaml:32]: just curious why is this removed? is the prediction inaccurate?

**Review by haibinxie — APPROVED** [review]

File: drivers/storage/portworx/testspec/prometheusRule.yaml (hunk 1 above)
Comment (piyush-nimbalkar) [reply drivers/storage/portworx/testspec/prometheusRule.yaml:32]: https://portworx.atlassian.net/browse/PD-904
